setup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2025 Google Inc. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Script that can be used by CI server for testing rules_webtesting builds.
+set -ex
+sudo apt -y update && sudo apt -y install libasound2 libdbus-glib-1-2 \
```

Comment:
the && is unnecessary if you're using `set -e`. Just run two command as it will exit early if the first fails.